package.json

```diff
@@ -114,6 +114,7 @@
     "postbuild": "node ./scripts/postbuild.js",
     "schema": "json2ts --input src/ng-package.schema.json --output src/ng-package.schema.ts",
     "release": "standard-version --releaseCommitMessageFormat 'release: cut v{{currentTag}}' --no-verify",
+    "postinstall": "yarn schema",
```

Comment:
Why is this needed?

Reply:
If you just just run `git clone` and `npm install`, if you open the project inside an IDe, you will get errors in all files that import the schema interface. This fixes that issue.